queries/faust/highlights.scm

```diff
@@ -0,0 +1,216 @@
+; Identifiers
+(identifier) @variable
+
+(process) @variable.builtin
+
+(parameters
+  (identifier)) @variable.parameter
+
+; Literals
+(_
+  filename: (string)) @string.special.path
+
+(function_names) @string.special
```

Comment:
Not `@function`?

Reply:
This is a language feature for declaring foreign functions. This node matches e.g. `asinhf|asinh|asinhl|asinfx` in
```
asinh = ffunction(float asinhf|asinh|asinhl|asinfx(float), <math.h>, "");
```
as `(function_names)`

Reply:
So… `@function`.

Reply:
I suppose so, it's a custom syntax for specifying a list of foreign function names by concatenating them with `|`. Happy to change this to `@function` if you think it matches better

Reply:
Pushed